x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/components/precision_slider/utils.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { STEP_DESCRIPTIONS } from './constants';
+
+export const getStepDescription = (precision: number) => {
+  // Precision steps are 1-11
+  // STEP_DESCRIPTIONS is indexed 0-10
+  const descriptionIndex = precision - 1;
+  return STEP_DESCRIPTIONS[descriptionIndex];
+};
```

Comment:
This seemed like the most testable approach but I'm open to other suggestions

Reply:
Just a thought... in your constants.ts:

```js
export const STEP_DESCRIPTIONS = [
  '', // The precision number we get from the API starts with 1 instead of 0
  STEP_O1_DESCRIPTION,
  STEP_O2_DESCRIPTION,
  STEP_03_DESCRIPTION,
  // ...etc
];
```

and then you can just do `STEP_DESCRIPTIONS[precision]` with no util

Reply:
So I was thinking about this but couldn't work out how I'd write tests for it, the function I could write tests for and then mock for the component.  Open to suggestions though.

Reply:
You wouldn't write tests for it, it would just work - that's the main advantage of that simpler approach vs a util fn. Just IMO, there isn't a need to write tests for basic language-level expectations like `someArray[someIndex]`. You also wouldn't need to mock`getStepDescription` at all.

Reply:
Sorry, just saw the re-request for review - Byron, were you implementing this & should I wait for that, or did you want me to QA as-is?

Reply:
https://github.com/elastic/kibana/pull/103015/commits/12537aea72b75dbb7c72e8830a033f328c7dbbe4 looks great!! ✨